modules/Bio/EnsEMBL/VariationMart/CreateMTMPTables.pm

```diff
@@ -32,7 +32,7 @@ sub param_defaults {
     'motif_exits'       => 1,
     'show_sams'         => 1,
     'show_pops'         => 1,
-    'scratch_dir'       => '/hps/scratch/flicek/ensembl',
+    'scratch_dir'       => catdir('/hps/nobackup/flicek/ensembl', $self->o('env_user'), $self->o('pipeline_name')),
```

Comment:
I haven't tried but I think `use File::Spec::Functions qw(catdir);` is required for `catdir`. Also I'm not sure about `$self->o('env_user')` being defined.

Reply:
Yup well spotted

Reply:
Wrong copy past from other config. Too hasty I was!